sdk/transform/transform.go

```diff
@@ -2,107 +2,93 @@ package transform
 
 import (
 	"bytes"
-	"reflect"
-	"strings"
+	"github.com/pkg/errors"
+	"google.golang.org/protobuf/encoding/protojson"
+	"io"
+	"os"
 	"text/template"
 
 	"github.com/aserto-dev/ds-load/sdk/common/js"
 	"github.com/aserto-dev/ds-load/sdk/common/msg"
-
-	v2 "github.com/aserto-dev/go-directory/aserto/directory/common/v2"
 )
 
-type Transformer struct {
-	MaxChunkSize int
+type Transform interface {
```

Comment:
Who uses this interface? Why do we need it?

Reply:
Removed the interface as it was not used